src/Compilers/Core/Portable/CommandLine/EditorConfig.cs

```diff
@@ -65,8 +71,7 @@ private EditorConfig(Section globalSection, ImmutableArray<Section> namedSection
         /// <summary>
         /// Gets whether this editorconfig is a the topmost editorconfig.
         /// </summary>
-        public bool IsRoot
-            => GlobalSection.Properties.TryGetValue("root", out string val) && val == "true";
+        public bool IsRoot => GlobalSection.Properties.TryGetValue("root", out string val) && val == "true";
 
```

Comment:
If the topmost editorconfig file in the project lacks this property, it should still be considered the root. How do we handle that?

Reply:
This comment is bad. `Root` should mean that no properties from higher editorconfigs should apply, not necessarily that there couldn't be editorconfigs higher in the directory tree. This work is tracked at https://github.com/dotnet/roslyn/issues/27644